server/subscriptions.go

```diff
@@ -276,3 +290,65 @@ func (p *Plugin) Unsubscribe(channelID string, repo string) error {
 
 	return nil
 }
+
+func (p *Plugin) UnsubscribeOrg(ctx context.Context, githubClient *github.Client, channelID, org string) error {
+	config := p.getConfiguration()
+
+	if org == "" {
+		return fmt.Errorf("Invalid organization")
+	}
+
+	_, org, _ = parseOwnerAndRepo(org, config.EnterpriseBaseURL)
+	if err := p.checkOrg(org); err != nil {
+		return err
+	}
+
+	subs, err := p.GetSubscriptions()
+	if err != nil {
+		return err
+	}
+
+	// Get all repos in an organization
+	listOrgOptions := github.RepositoryListByOrgOptions{
+		Type: "all",
+		ListOptions: github.ListOptions{
+			Page: 1,
+		},
+	}
+
+	orgRepos := map[string]github.Repository{}
+	fetchingRepos := true
+	for fetchingRepos {
```

Comment:
Please see my comment on the other PR: https://github.com/mattermost/mattermost-plugin-github/pull/56#discussion_r262089022